### PR TITLE
Feature/add dotenv

### PR DESCRIPTION
**Pull request by robinheinze**
I went with `react-native-dotenv` over `react-native-config` because the latter had tons of issues installing, apparently due to incompatibility with Cocoapods? There were some hacky workarounds, but I still couldn't get it to work. The install process for `react-native-dotenv` was much smoother, and did exactly what we needed. 

Also added a couple lines to the README to direct users to the right place if they need secret tokens.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,10 @@
 - for Android
   - run `react-native run-android`
 
+# Secrets
+
+If you are working on the AWS AppSync talk discussion feature of the app, you will need a `.env` file with the proper API token. Please ping @jamon in the #chainreact channel of the [Infinite Red community Slack](https://community.infinite.red) and we'll hook you up!
```
Comment (jamonholmgren) [inline README.md]: Unfortunately, we haven't set up https yet for that landing page. :shame:

```suggestion
If you are working on the AWS AppSync talk discussion feature of the app, you will need a `.env` file with the proper API token. Please ping @jamon in the #chainreact channel of the [Infinite Red community Slack](http://community.infinite.red) and we'll hook you up!
```

**Comment by ryanlntn** [comment]
You could update the setup script to write a blank .env file too. Or just include a .env.sample that isn’t gitignored.

**Comment by robinheinze** [comment]
Took Ryan's suggestion and made the setup script create a blank `.env` file with a fake token. Unfortunately, the app won't run at all without the token in there because graphql blows up, so anybody doing any work on it will need to get it.

**Comment by jamonholmgren** [comment]
Excellent!

**Pull request merged**